lib/module.js

```diff
@@ -520,10 +524,11 @@ Module.prototype.load = function(filename) {
 
 // Loads a module at the given file path. Returns that module's
 // `exports` property.
-Module.prototype.require = function(path) {
+Module.prototype.require = function(path, opts) {
   assert(path, 'missing path');
   assert(typeof path === 'string', 'path must be a string');
-  return Module._load(path, this, /* isMain */ false);
+  assert(opts === undefined || typeof opts === 'object', 'opts must be an object');
```

Comment:
Also, I'm really wondering if checking the argument here makes any sense at all. The type check at `_load` might be just enough.